alexandria/storage/src/list.cairo

```diff
@@ -10,7 +11,7 @@ use traits::{Default, DivRem, IndexView, Into, TryInto};
 
 const POW2_8: u32 = 256; // 2^8
 
-#[derive(Drop)]
+#[derive(Drop, Copy)]
```

Comment:
Is adding `Copy` necessary? What does it do? 🤔

Reply:
I am not very experienced with Copy but I had to add it since I copied value from a ref (self) to an array (reference already moved issue).
Switching to snapshot made it unecessary :+1: